examples/bri-3/src/bri/transactions/agents/transactions.agent.ts

```diff
@@ -58,7 +58,17 @@ export class TransactionAgent {
       throw new BadRequestException(UPDATE_WRONG_STATUS_ERR_MESSAGE);
     }
 
-    return transactionToUpdate;
+    return new Transaction(
```

Comment:
why is this needed when there is already a mapping from prisma model to domain object in the storage agent?

Reply:
this is needed till the time https://github.com/eea-oasis/baseline/issues/553 is resolved, or we have auto-mapper in place, whatever comes first. This will be changed once we have that in place